test/success-tests/SimpleMMA.t.sol

```diff
@@ -47,12 +50,15 @@ contract MMA is Setup {
                 msgId = msgIdLogs[i].topics[1];
             }
         }
-        console.logBytes32(msgId);
 
         vm.selectFork(fork[137]);
 
-        /// execute message received
+        /// execute the message and move it to governance timelock contract
         MultiMessageReceiver(contractAddress[137][bytes("MMA_RECEIVER")]).executeMessage(msgId);
+
+        /// increment the time by 2 day (delay time)
+        vm.warp(block.timestamp + 1 days);
+        GovernanceTimelock(contractAddress[137][bytes("TIMELOCK")]).executeTransaction(1);
```

Comment:
This doesn't look quite right to me. I think this test should have failed, given you are only adding a day and not the required delay (2 days).

Reply:
I suggest we also add tests around the failing conditions, so we can catch related issues.

Reply:
suggest we also add tests for the grace period validation